dropshot/tests/fail/duplicate_argument_type.stderr

```diff
@@ -0,0 +1,7 @@
+error[E0119]: conflicting implementations of trait `handler_argument_for_endpoint_with_duplicate_raw_requests` for type `hyper::Request<hyper::Body>`:
+  --> $DIR/duplicate_argument_type.rs:29:5
+   |
+28 |     _: &mut Request<Body>,
+   |     - first implementation here
+29 |     _: &mut SneakyBody
+   |     ^ conflicting implementation for `hyper::Request<hyper::Body>`
```

Comment:
I'm not totally happy with this error message. The trick I got from the static-assert crate for making sure no types are the same is to implement a trait for all of them, and then redundant traits will cause a compile time error. The best I could to make the compile-time error mean something was to give the trait the name above

`handler_argument_for_${function_name}`

It gets the point across, but bleh